tests/network/network_service/conftest.py

```
@@ -13,6 +15,39 @@
 from utilities.network import compose_cloud_init_data_dict
 from utilities.virt import VirtualMachineForTests, fedora_vm_body
 
+LOGGER = logging.getLogger(__name__)
+
+
+def _create_virtctl_expose_service(
```

Ruff (0.14.13)

Missing return type annotation for private function `_create_virtctl_expose_service`

(ANN202)